examples/AspNetCore/Program.cs

```diff
@@ -121,34 +122,25 @@
                 builder.AddConsoleExporter();
                 break;
         }
-    });
-
-// Clear default logging providers used by WebApplication host.
-appBuilder.Logging.ClearProviders();
-
-// Configure OpenTelemetry Logging.
-appBuilder.Logging.AddOpenTelemetry(options =>
-{
-    // Note: See appsettings.json Logging:OpenTelemetry section for configuration.
-
-    var resourceBuilder = ResourceBuilder.CreateDefault();
-    configureResource(resourceBuilder);
-    options.SetResourceBuilder(resourceBuilder);
```

Comment:
❤️ seeing common Resource across all 3!

And also ❤️ fixing the awkwardness of calling methods like `AddExporter` on a supposedly pure "Options" class!